tests/docker-scripts/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.8'
+
+services:
+  dhc-server:
+    image: ghcr.io/deephaven/server:${VERSION:-latest}
```

Comment:
Good catch. I agree it doesn't make too much sense to test with locally built versions because that test would fail in CI if the core change isn't merged first